js/demo/lazy-loading.js

```diff
@@ -105,8 +93,8 @@ var EffecktLazyLoading = {
         self._scrollPage();
       }, 1000 / 60);
     }
-  },
+  }
```

Comment:
@arthurvr Sure alright, will close this PR and open another one with the changes mentioned above. Also, once we add a jshint file, we should open an issue for jshint compliance.